internal/wasm/instruction/variable.go

```diff
@@ -36,3 +36,19 @@ func (SetLocal) Op() opcode.Opcode {
 func (i SetLocal) ImmediateArgs() []interface{} {
 	return []interface{}{i.Index}
 }
+
+// TeeLocal represents the WASM tee_local instruction.
+type TeeLocal struct {
```

Comment:
👍 I was meaning to go through and add this and replace any of the unnecessary `SetLocal => GetLocal` combinations.

Reply:
Awked my way through, replaced the obvious ones (on subsequent lines).